src/containers/repository.py

```diff
@@ -0,0 +1,23 @@
+from typing import List
+
+
+class Repository:
+    def __init__(self):
+        self.organization_name: str = ""
+        self.repository_name: str = ""
+        self.query_labels: List[str] = []
+
+    def load_from_json(self, repository):
```

Comment:
Here is a missing type for the repository parameter.

Reply:
https://github.com/AbsaOSS/living-doc-generator/pull/16#discussion_r1680641697